packages/suite/src/components/wallet/CoinmarketBuyProviderInfo/index.tsx

```diff
@@ -9,10 +9,17 @@ const Wrapper = styled.div`
     align-items: center;
 `;
 
+const Bg = styled.div<{ color: string }>`
```

Comment:
not really important, maybe just my OCD but we use full-word naming of the tags everywhere :)